src/test/ui/loops/loop-break-value.stderr

```diff
@@ -90,10 +90,10 @@ error[E0308]: mismatched types
   --> $DIR/loop-break-value.rs:4:31
    |
 LL |     let val: ! = loop { break break; };
-   |                               ^^^^^ expected (), found !
+   |                               ^^^^^ expected !, found ()
    |
-   = note: expected type `()`
-              found type `!`
+   = note: expected type `!`
+              found type `()`
```

Comment:
@Centril Re: suggestion when encountering an expectation of `break !`, I feel that giving it too much consideration for a suggestion is not necessary, as I can't think of a non-degenerate construct that would cause such a case, as shown by this test.